lib/config-loader.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+var cloneDeep = require('lodash').cloneDeep;
```

Comment:
> lodash has a lot of useful utility functions, might make more sense to require it on its own so that someone can use other lodash functions without changing this line.

I am just following the style seen in existing code, see e.g. https://github.com/strongloop/loopback-boot/blob/296c38568cc13a7d4fc80c1aec313eef801d06ae/lib/compiler.js#L7

> another note on lodash, we had previous goals to shrink bundle size, would this be going against that, 

Bundle size is important, thank you for raising this concern. The browser bundle contains only the source code from `lib/executor.js`, config loader is executed during compilation, therefore it's ok to use lodash here.